client/my-sites/plugins/plugin-details-CTA/index.jsx

```diff
@@ -99,6 +105,14 @@ const PluginDetailsCTA = ( { plugin, isPlaceholder } ) => {
 		getEligibility( state, selectedSite?.ID )
 	);
 
+	const getUpgradeToBusinessHRef = useCallback( () => {
+		const pluginsPlansPageFlag = isEnabled( 'plugins-plans-page' );
+
```

Comment:
That feature flag comes from using the same logic than the upsell banner displayed in the main `/plugins/<site>` page. This is the condition https://github.com/Automattic/wp-calypso/blob/94eac9ba493dad297c2e8cb6205380ad9a6afe70/client/my-sites/plugins/plugins-discovery-page/upgrade-nudge.jsx#L151

I thought about moving that condition to a `utils` common function but I did not do it because the final logic is still to be confirmed